spec/index.html

```diff
@@ -199,43 +209,59 @@ <h3>Example</h3>
-    is US-ASCII. The <code>charset</code> parameter <em class="rfc2119" title="should">should</em> be used in conjunction with SPARQL Results; UTF-8 is recommended: <code>text/csv; charset=utf-8</code> and
-    <code>text/tab-separated-values; charset=utf-8</code>.</p>
-    <p>The end-of-line in CSV is <code>CRLF</code> i.e. Unicode codepoints 13 (0x0D) and 10 (0x0A).</p>
-    <p>The end-of-line in TSV is <code>EOL</code> i.e. Unicode codepoint 10 (0x0A).</p>
-    <p>Applications reading these formats are advised to cope with both CRLF and LF as end of line markers and not rely on conformance to the formal specifications.</p>
+    <p>The SPARQL result formats described here conform to the formal specifications of the relevant formats, 
+      Comma Separated values (CSV) [<cite><a class="bibref" rel="biblioentry" href="#bib-RFC4180">RFC4180</a></cite>] 
+      and Tab Separated Value (TSV) [<cite><a class="bibref" rel="biblioentry" href="#bib-IANA-TSV">IANA-TSV</a></cite>].</p>
+    <p>Systems providing these formats should note that the content types are <code>text/csv</code> for CSV and 
+      <code>text/tab-separated-values</code> for TSV. Being <code>text/*</code>, the default character set
+      is US-ASCII. The <code>charset</code> parameter <em class="rfc2119" title="should">should</em> be used in
+      conjunction with SPARQL Results; UTF-8 is recommended; giving us <code>text/csv; charset=utf-8</code> and
+      <code>text/tab-separated-values; charset=utf-8</code>.</p>
+    <p>The end-of-line in CSV is <code>CRLF</code>, i.e., Unicode code points <code>U+0013</code> (<code>0x0D</code>) and <code>U+0010</code> (<code>0x0A</code>).</p>
+    <p>The end-of-line in TSV is <code>EOL</code>, i.e., Unicode code point <code>U+0010</code> (<code>0x0A</code>).</p>
+    <p>Applications reading these formats are advised to cope with both CRLF and LF as end of line markers and
+      not rely on conformance to the formal specifications.</p>
     </section>
     
     <section id="csv-table">
-    <h2>CSV - Comma Separated values</h2>
-    <p>In the SPARQL Results CSV Format, the results table is serialized as one line listing the variables in the results, using the CSV header line, followed by one line for each query solution (a
-    line may end up split by newlines in the data). Values in the results are strings, for URIs, literals and blank nodes, together with numbers when the literals are of numeric XSD datatype.</p>
+    <h2>CSV — Comma Separated values</h2>
+    <p>In the SPARQL Results CSV Format, the results table is serialized as one line listing the variables in
+      the results, using the CSV header line, followed by one line for each query solution. (Note: a line may end up
+      split by newlines in the data). Values in the results are strings for URIs, literals, and blank nodes, together
+      with numbers when the literals are of numeric XSD datatype.</p>
```

Comment:
```suggestion
      split by newlines in the data). Values in the results are:</p>
      <ul>
        <li>strings for URIs, literals of non-numeric XSD datatype, and blank nodes</li>
        <li>numbers for literals of numeric XSD datatype</li>
      </ul>
```

Reply:
I think this one is less likely, but possible.
```suggestion
      split by newlines in the data). Values in the results are:</p>
      <ul>
        <li>strings for URIs</li>
        <li>literals of non-numeric XSD datatype</li>
        <li>blank nodes</li>
        <li>numbers for literals of numeric XSD datatype</li>
      </ul>
```

Reply:
And of course, there's one more possibility...
```suggestion
      split by newlines in the data). Values in the results are:</p>
      <ul>
        <li>strings for 
          <ul>
            <li>URIs</li>
            <li>literals of non-numeric XSD datatype</li>
            <li>blank nodes</li>
          </ul>
        </li>
        <li>numbers for literals of numeric XSD datatype</li>
      </ul>
```

Reply:
Version 1 (`<li>strings for URIs, literals of non-numeric XSD datatype, and blank nodes</li>`) looks best to me. 

Version 3 is OK but a bit heavy in context. Version 2 would have to say "lexical form" not "literal".

Reply:
V1 is framed more like CSV - string and numbers. "strings for" then a list of 3 items.

V2 says "strings for URIs" but not "strings for literals ..." - it's "values are literals"

Reply:
Given everything in this thread, including my own misunderstandings of what was meant by the original, I think only my third suggestion communicates what was intended without likelihood of being misunderstood.

Reply:
> s/blank node/blank nodes labels/

Applied to all three variations.

> Either [V1 or V3] is acceptable and better than the current text.

I prefer V3, because it eliminates potential misunderstanding _that I experienced with V1_, and I am applying it now, as I believe we have concurrence on its acceptability.

Reply:
> s/blank node/blank nodes labels/
> 
> V1 and V3 are the same, and differ only be expanding the sentence into a list. In the current context, I prefer the shorter form because CSV as we use it, has two forms: strings and numbers.
> 
> Either is acceptable and better than the current text.

RDF Concepts defines “blank nod identifier” rather than “label”, but is so commonly misappropriated that maybe it should be defined as a synonym.